examples/onthefly.js

```diff
@@ -43,7 +43,7 @@ function getHttpsOptionsFromDisk(domain, callback) {
     } else {
       fs.readFile(certificatesFolder + domain + '/key.pem', function(err2, keyData) {
         if (err1) {
```

Comment:
In any case, please also see https://github.com/Daplie/node-letsencrypt which is now actively maintained.